submissions/kirill8210/popup-task/main.css

```diff
@@ -0,0 +1,265 @@
+html{
+    box-sizing: border-box;
+}
+
+*, *::before, *::after{
+    box-sizing: inherit;
+}
+
+body{
+    font-family: Roboto, sans-serif;
+    line-height: 1.3;
+    font-weight: 400;
+}
+
+img{
+    max-width: 100%;
+    height: auto;
+}
+
+button{
+    cursor: pointer;
+    padding: 0;
+}
+
+a{
+    text-decoration: none;
+    color: inherit;
+}
+
+ul{
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+h1, h2, h3, p{
+    margin: 0;
+}
+
+.container{
+    min-height: 100vh;
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: 0 30px;
+}
+
+.header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.menu{
+    display: flex;
+    align-items: center;
+    padding: 10px 0;
+    position: relative;
+}
+
+.menu_li{
+    display: flex;
+    align-items: center;
+    margin-left: 10px;
+}
+
+.menu_li{
+    border: 1px solid transparent;
+    height: 35px;
+}
+
+.menu_li > a:hover{
+    border: 1px solid #e5e5e5;
+    padding: 5px 6px;
+}
+
+.menu_li a:focus {
+    border: 2px solid #a6c8ff;
+}
+
+.menu_li > a:hover:focus {
```

Comment:
Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)